mypy/test/data.py

```diff
@@ -166,6 +167,10 @@ def _item_fail(msg: str) -> NoReturn:
                         version = tuple(int(x) for x in version_str.split("."))
                     except ValueError:
                         _item_fail(f"{version_str!r} is not a valid python version")
+                    if version < defaults.PYTHON3_VERSION_MIN:
```

Comment:
Should this be `defaults.PYTHON3_VERSION` rather than `defaults.PYTHON3_VERSION_MIN`?

https://github.com/python/mypy/blob/cab8c674ebef944498ae4e9a89cd9c77316656ec/mypy/defaults.py#L8-L15

Reply:
Hmmm, now that you're saying, yes, because this is affected by the Python version we're running on, not `--python-version`.